server/perform_search.py

```diff
@@ -15,10 +15,12 @@ def get_similar_images(self, keyword, semantic_query, pixabay_max, top_k):
         :param top_k: Top-k images to return.
         :return: List of top-k PIL images.
```

Comment:
The return part and the short description need to be changed then.

Reply:
sure! I have just commited this change, so please check it out!